Error building 4.0 on FreeBSD armv7

I was trying to build 4.0 version of ntopng on FreeBSD 12 armv7 (amd64 built with no issues) and got following error:
```
src/ParsedFlow.cpp:130:14: error: invalid operands to binary expression ('lua_Number' (aka 'double') and 'double')
          version = htons(lua_tonumber(L, -1));
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/netinet/in.h:120:18: note: expanded from macro 'htons'
#define htons(x)        __htons(x)
                        ^~~~~~~~~~
/usr/include/machine/endian.h:74:28: note: expanded from macro '__htons'
#define __htons(x)        (__bswap16(x))
                           ^~~~~~~~~~~~
/usr/include/machine/endian.h:129:6: note: expanded from macro '__bswap16'
     __bswap16_constant(x) :                    \
     ^~~~~~~~~~~~~~~~~~~~~
/usr/include/machine/endian.h:124:12: note: expanded from macro '__bswap16_constant'
    ((((x) & 0xff00) >> 8) |            \
       ~~~ ^ ~~~~~~
```

--- a/src/ParsedFlow.cpp
+++ b/src/ParsedFlow.cpp
@@ -127,17 +127,17 @@ void ParsedFlow::fromLua(lua_State *L, int index) {
 	if(!strcmp(key, "vlan_id"))
 	  vlan_id = lua_tonumber(L, -1);
 	else if(!strcmp(key, "version"))
-	  version = htons(lua_tonumber(L, -1));
+	  version = htons(lua_tointeger(L, -1));
 	else if(!strcmp(key, "src_port"))
-	  src_port = htons(lua_tonumber(L, -1));
+	  src_port = htons(lua_tointeger(L, -1));
 	else if(!strcmp(key, "dst_port"))
-	  dst_port = htons(lua_tonumber(L, -1));
+	  dst_port = htons(lua_tointeger(L, -1));
 	else if(!strcmp(key, "l4_proto"))
 	  l4_proto = lua_tonumber(L, -1);
 	else if(!strcmp(key, "tcp_flags"))
-	  tcp.tcp_flags = htons(lua_tonumber(L, -1));
+	  tcp.tcp_flags = htons(lua_tointeger(L, -1));
 	else if(!strcmp(key, "direction"))
-	  direction = htons(lua_tonumber(L, -1));
+	  direction = htons(lua_tointeger(L, -1));
 	else if(!strcmp(key, "first_switched"))
 	  first_switched = lua_tonumber(L, -1);
 	else if(!strcmp(key, "last_switched"))

